execution/native/permissions.go

```diff
@@ -10,75 +10,75 @@ import (
 )
 
 var Permissions = New().MustContract("Permissions",
-	`* acmstate.ReaderWriter for managing Secure Native authorizations.
```

Comment:
Changes in this file are unrelated to this PR, in the future we should keep changes to a minimum.

Reply:
Not unrelated... I needed to regenerate the permissions contract to make the deploy tests work and the comments needed to match to make the solidity compiler happy.

Reply:
I'll admit there are a few changes here that aren't strictly needed for the fix but given deploy tests were bork you get a big of a rag tag bunch and I CBA to make secondary PRs for stuff like logging quality of life improvements...